### ISSUE TITLE
Vimium doesn't work on startup page with startup option 'Continue where you left off'

### ISSUE THREAD

**Issue by asdf123101** [philc/vimium#2990]
### Chrome/Chromium
Version 65.0.3325.146 (Official Build) (64-bit)
### Vimuim
Version 1.63
### OS
Linux Mint 18.3 and Arch Linux
---
The startup page is just a regular webpage. Vimum is suppose to work on this page because it worked before I closed Chrome. However, at startup, Vimium does not work on this particular page while works on other pages open by chrome. 

This behavior started to happen recently, like last week.

**Comment by tuxinaut** [comment]
Same under `Firefox`

**Comment by smblott-github** [comment]
Yes.  This is a race condition between Vimium's front-end JS and the JS on the background page.  The front-end JS runs before the background page is listening, so initialisation fails.  We should fix this.

**Comment by carsonalh** [comment]
@smblott-github Please can you fix this, it's really annoying having to have one's flow broken by not using the extension just as the page starts up. Would really appreciate if you could get around to fixing this now.

**Comment by MuGi-131** [comment]
This is not a solution but a way to work around this issue.
I used custom chrome startup page it helps me to keep the flow.
Cheers!

**Comment by copiummax** [comment]
@smblott-github any updates :D ?
this bug is quite annoying since i restart my browser often

**Comment by daverbk** [comment]
a bit strange comment from my side, but I hope it might be useful to somebody visiting this issue)
this whole thing with the stratup page kind of breaks the experience for me so I've replaced all the possible ways it might appear

1. On startup -> Open a specific page
2. Vimium -> New tab URL 
3. `New tab redirect` extension for chrome

**Comment by USAMAWIZARD** [comment]
the start page which is by default it comes under chrome://newtab , chrome does not allows extentions to run on these pages , https://github.com/philc/vimium/pull/4583 this pr should allow users to do that by enabling some flag in chrome
